### ISSUE TITLE
MonoGame Pipeline Tool problem with .NET Core Nuget Extensions

### ISSUE THREAD

**Issue by WardBenjamin** [MonoGame/MonoGame#6301]
Since Nuget packages are stored in `~/.nuget/packages` instead of the `./packages` directory for .NET Core projects, the MGCB needs to reference a different path for any Pipeline extensions that are delivered via Nuget. I ran into this with https://github.com/craftworkgames/MonoGame.Extended/issues/495, but it should be equally applicable to other extensions. 

```
/reference:/home/wardp/.nuget/packages/monogame.extended.content.pipeline/1.0.617/tools/MonoGame.Extended.Content.Pipeline.dll
```

While the changed path is simple enough to hard-code, MGCB should provide access to the `$HOME` environment variable so that this would be portable across installations.

- MonoGame Pipeline Tool 3.6.0.1625
- Arch Linux (Kernel: 4.14.35-1-lts)
- DesktopGL.Core (thanks @cra0zy, works great with Rider)

**Comment by cra0zy** [comment]
Yea, this has nothing do with .NET Core, this is just how nuget works with the new .csproj format. I have it on my TODO list to provide Pipeline Tool with NuGet package manager so you can reference the required nugets from it directly.

**Comment by WardBenjamin** [comment]
@cra0zy do you have any update on this? The only solution that I've come across is to manually copy or symlink NuGet packages into a folder local to the MGCB file.

**Comment by cra0zy** [comment]
None so far, once my exams end I'll see if I can add nuget reference support.

**Comment by tomizechsterson** [comment]
We just got burned on this one too. It'd be awesome if this could get fixed.

@WardBenjamin , how exactly did you solve it by manually copying the dll? I get `System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information` when I try and do that. Even if I copy all 3 dll's from the nuget folder

**Comment by Apostolique** [comment]
@tomizechsterson This is what I do. I copy the dll to my project's root so I can reference it using the content pipeline.

**Comment by Apostolique** [comment]
@tomizechsterson Indeed, this is pretty confusing. Are you on the [MonoGame Discord](https://discord.gg/N9t26Uv)? [![Discord](https://img.shields.io/discord/355231098122272778.svg)](https://discord.gg/N9t26Uv)

**Comment by tomizechsterson** [comment]
@Apostolique I wasn't. I am now, though :)
